README.md

```diff
@@ -1,64 +1,31 @@
 linter-foodcritic
 =========================
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides an interface to [foodcritic](https://github.com/AtomLinter/linter-foodcritic). It will be used with files that have the “Ruby” syntax.
+This linter plugin for [Linter](https://github.com/atom-community/linter)
```

Comment:
It would've been nice if it was `This is a [linter][linter] provider for [foodcritic][foodcritic]`